typegate/tests/runtimes/grpc/grpc_test.ts

```diff
@@ -6,6 +6,20 @@ test("Grpc runtime", async (t) => {
   const e = await t.pythonFile("runtimes/grpc/grpc.py");
 
   await t.should("works", async () => {
+    new Deno.Command(
+      "./typegate/tests/runtimes/grpc/grpc_server/start.sh",
+      {
+        stdout: "null",
+      },
+    ).spawn();
+
+    const sleep = new Deno.Command("/bin/sleep", {
+      args: ["5"],
+      stdout: "null",
+    }).spawn();
+
+    await sleep.status;
```

Comment:
you can use `sleep` from `utils.ts` in test

Reply:
Thank you, I didn't notice that helper.